### PR TITLE
DON'T MERGE [METAED-1494] Add edfi-model-5.0

### PR DESCRIPTION
**Pull request by jleiva-gap**
Install  edfi-model-5.0 to replace prerelease package uninstalled.

### CODE REVIEW

**Comment by bradbanister** [comment]
@jleiva-gap  I don't understand why this is necessary

**Comment by jleiva-gap** [comment]
This adds the dependency to ed-fi-model 5.0 instead of 5.0-pre.1. When we removed the prerelease package we didn't have the version 5.0.0

File: package.json [hunk 1]
```diff
@@ -174,6 +173,7 @@
 		"vscode-uri": "^3.0.6"
 	},
 	"devDependencies": {
+		"@edfi/ed-fi-model-5.0": "3.0.0",
```
Comment (stephenfuqua) [inline package.json:176]: @jleiva-gap looking at the diff here, it looks like you just moved the 5.0 dependency from `dependencies` to `devDependencies`, whereas the original line 146 was already correct.

Comment (jleiva-gap) [reply package.json:176]: Oh, I see, It was moved when I run the npm install, so I think we can close this PR, the repo is already updated, I didn't need to add anything else to this repo.

**Pull request closed without merging**